js/src/modals/AddContract/addContract.js

```diff
@@ -145,6 +145,7 @@ export default class AddContract extends Component {
       api.personal.setAccountMeta(address, {
         contract: true,
         deleted: false,
+        timestamp: Date.now(),
```

Comment:
Could we add this to the addressbook & accounts as well - I know you won't use it currently, but I'd rather have the data in there for the sake of consistency & possible future use.